Fix the postgresql database not being preserved

mikedep333 opened the pull request:
for docker-compose.

fixes: #468

Also fix the transfer process.

And fix a CI issue with podman-compose by limiting us to 1.0.3 and its healthcheck syntax.

mikedep333 committed:
Fix the postgresql database not being preserved

for docker-compose.

fixes: #468

mikedep333 commented:
To fix existing systems:
```
curl https://raw.githubusercontent.com/alaub81/backup_docker_scripts/main/backup-docker-volume.sh -o backup-docker.volume.sh
sudo docker inspect compose_postgres_1
# Observe what the long random hexadecimal Name of the ephemeral volume is, under Mounts.
# Optional: Edit backup-docker.volume.sh so that VOLUME is only set to the specific volume Name
sudo docker-compose stop
sudo bash backup-docker.volume.sh
sudo ls -latr /backup/
# Verify that there is a tarball  with the Name of the volume, with a timestamp hyphenated after it.
sudo docker-compose down
# This next command creates the correct volume pg_datadev. The filepath must be specified
sudo docker run --rm -v /backup/volumes:/backup -v pg_datadev:/data --name restore alpine sh -c "cd /data && /bin/tar -xvf /backup/c532edb6e667574d73b6f6efc75f2e2bc39f1cce0bff3b417a6b1fec35e39b87-202305051806.tar.gz"
sudo docker-compose up
```

bmbouter reviewed (approved):
Thanks, please merge once CI passes

mikedep333 commented on images/compose/docker-compose.yml:
```diff
@@ -11,7 +11,7 @@ services:
       POSTGRES_HOST_AUTH_METHOD: 'scram-sha-256'
     volumes:
-      - "pg_data:/var/lib/postgresql"
+      - "pg_data:/var/lib/postgresql/data"
```
Re-Add the Z for SELinux.

mikedep333 committed:
Fix transferring from s6 images to compose

[noissue]

mikedep333 committed:
Limit podman-compose to 1.0.3

Similar to how Ubuntu 22.04 has podman 3.4.4
from Dec 2021 also.

Requires using CMD-SHELL healthcheck implicitly to avoid a parsing error.

Fixes "error generating dependency graph for container"

[noissue]
